### ISSUE TITLE
Include toolkit slides in master deck

### ISSUE THREAD

**Issue by tclayson** [kennethkutyn-optimizely/deckbuilder#19]
There is a toolkit slide which can be found in this presentation: https://docs.google.com/presentation/d/1VgeqqVYPWQ1_gOlWXP7YgupuPqkaFUcbnAR9k04PoYw/edit?usp=sharing

There are also some "web vs full stack" slides which we should include if possible.

**Comment by kennethkutyn-optimizely** [comment]
Is this the right deck link?

**Comment by tclayson** [comment]
Yeah the "web" "fullstack" and "shared platform" slide - slide 2 in that link. :)
